code/modules/mob/living/living_health_procs.dm

```diff
@@ -1,6 +1,20 @@
 
 //procs directly related to mob health
-
+/mob/living/Life(delta_time)
```

Comment:
couldn't this be done with signals instead of hooking *into the Life() of every single `living` mob*?